modeshape-shell/src/main/java/org/modeshape/example/embedded/ModeShapeExample.java

```diff
@@ -0,0 +1,94 @@
+/*
+ * ModeShape (http://www.modeshape.org)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * 	http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.modeshape.example.embedded;
+
+import java.net.URL;
+import javax.jcr.Node;
+import javax.jcr.Repository;
+import javax.jcr.Session;
+import org.modeshape.common.collection.Problems;
+import org.modeshape.jcr.ModeShapeEngine;
+import org.modeshape.jcr.RepositoryConfiguration;
+
+public class ModeShapeExample {
```

Comment:
Need a good description of what this examples shows and how it goes about doing that.